client/ayon_royalrender/plugins/publish/create_maya_cache_royalrender_job.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""Submitting render job to RoyalRender."""
+import os
+
+from maya import cmds
+from maya.OpenMaya import MGlobal  # noqa: F401
+
+from ayon_royalrender import lib
+
+
+class CreateMayaCacheRoyalRenderJob(lib.BaseCreateRoyalRenderJob):
+    label = "Create Maya Cache job in RR"
+    hosts = ["maya"]
+    families = ["pointcache"]
+
+    def update_job_with_host_specific(self, instance, job):
+        job.Software = "Maya"
+        job.Renderer = "RemotePublish"
+        job.Version = "{0:.2f}".format(MGlobal.apiVersion() / 10000)
+        job.CustomScriptFile = "<rrLocalRenderScripts>/ayon_remote_publish.py"
+        workspace = instance.context.data["workspaceDir"]
+        job.SceneDatabaseDir = workspace
+        job.rrEnvList += f"~~~INSTANCE_IDS={instance.data['instance_id']}"
+
+        return job
+
+    def process(self, instance):
+        """Plugin entry point."""
+        super(CreateMayaCacheRoyalRenderJob, self).process(instance)
+
+        if not instance.data.get("farm"):
+                self.log.info("Skipping local instance.")
+                return
```

Comment:
Over-indented? Is the linter missing? And how about the linting on this repo. @antirotor do you know?
```suggestion
            self.log.info("Skipping local instance.")
            return
```